code/datums/mind.dm

```diff
@@ -109,6 +109,8 @@
 			output += "<B>Objective #[obj_count]</B>: [objective.explanation_text]"
 			obj_count++
 
+		if(ambitions)
+			output += "<HR><B>Ambitions:</B> [ambitions]<br>"

```

Comment:
On Bay this one is one (un)indentation further to the left.
As is, you'll now only see your ambitions if you also have objectives.